lbuild/node.py

```diff
@@ -246,15 +263,18 @@ def submodules(self):
     def configurations(self):
         return self._findall(self.Type.CONFIG, depth=2)
 
+    @property
+    def collectors(self):
+        return self._findall(self.Type.COLLECTOR, depth=2)
+
     @property
     def repository(self):
         return self._repository
 
     @property
     def dependencies(self):
-        if not self._dependencies_resolved:
-            self._resolve_dependencies()
```

Comment:
Why was the check beforehand removed?

Reply:
`self._resolve_dependencies()` already checks this itself, so you can just call it.